forc-util/Cargo.toml

```diff
@@ -27,6 +27,7 @@ sway-core = { version = "0.51.1", path = "../sway-core" }
 sway-error = { version = "0.51.1", path = "../sway-error" }
 sway-types = { version = "0.51.1", path = "../sway-types" }
 sway-utils = { version = "0.51.1", path = "../sway-utils" }
+sysinfo = "0.30.5"
```

Comment:
Unfortunately, the sysinfo crate is problematic for macos builds. See here #4737

Perhaps it's not an issue anymore but it was causing the nix builds to break. Might be worth looking into either clarifying this isn't the case anymore or finding an alternative approach.